### ISSUE TITLE
Sally task - check p values

### ISSUE THREAD

**Issue by sadamowi** [m-orton/Evolutionary-Rates-Analysis-Pipeline#20]
I am opening this issue to remind me to check the binomial and Wilcoxon tests, as requested. I will close this issue after I have done that.

**Comment by sadamowi** [comment]
I have checked the p-values for your binomial test. These are correctly calculated based upon your number of successes and failures for each class and overall. I still need to check the Wilcoxon test.

**Comment by sadamowi** [comment]
Hi Jacqueline and Matt,

I see that this older issue is still open. Is this still something I should check, or have you checked this Jacqueline? (Thanks for confirming. I saw you two discussing the lines of code for the stats tests in another thread.)

Cheers,
Sally

**Comment by sadamowi** [comment]
Hi Matt and Jacqueline,

I have checked the p-value calculator, using the outputted results for Mesostigmata. I used the outputted pairing results (from Server Run Jan 30th) to recalculate the p-value for the binomial test and the Wilcoxon test. I got the same results as were in the Pval tab in the Results Excel sheet. So, I think this test is being performed correctly, and so I will close this issue.

Cheers,
Sally
